aFWall/src/main/java/dev/ukanth/ufirewall/util/LogNetUtil.java

```diff
@@ -0,0 +1,217 @@
+package dev.ukanth.ufirewall.util;
+
+import android.content.ClipData;
+import android.content.ClipboardManager;
+import android.content.Context;
+import android.os.AsyncTask;
+import android.support.annotation.NonNull;
+
+import com.afollestad.materialdialogs.DialogAction;
+import com.afollestad.materialdialogs.MaterialDialog;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+import dev.ukanth.ufirewall.Api;
+import dev.ukanth.ufirewall.R;
+
+/**
+ * This file was created to simplify Network Function in AfWall+ log system
+ * Created by vzool on 1/20/17.
+ */
+
+public class LogNetUtil {
+
+    private static final String PING_CMD = "/system/bin/ping -q -n -w 1 -c 1 %s";
```

Comment:
Aha, yes it may happen...
What about make BusyBox be bundled with the app apk, because I think that many functionality will be added depends on the BusyBox.
What is your suggest?